packages/nimble-components/src/card/tests/card.stories.ts

```diff
@@ -0,0 +1,55 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { standardPadding } from '../../theme-provider/design-tokens';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../utilities/tests/storybook';
+import { listOptionTag } from '../../list-option';
+import { numberFieldTag } from '../../number-field';
+import { selectTag } from '../../select';
+import { cardTag } from '..';
+
+const overviewText = `The \`nimble-card\` is a container that is designed to contain arbitrary content that is specified by a client
+application. The \`nimble-card\` is intended for grouping related content.`;
+
+const metadata: Meta = {
+    title: 'Incubating/Card',
+    tags: ['autodocs'],
+    parameters: {
+        docs: {
+            description: {
+                component: overviewText
+            }
+        },
+        actions: {}
+    },
+    render: createUserSelectedThemeStory(html`
+    ${incubatingWarning({
+        componentName: 'card',
+        statusLink: 'https://github.com/ni/nimble/issues/296'
+    })}
+        <style>
+            .card-content {
+                display: flex;
+                flex-direction: column;
+                gap: var(${standardPadding.cssCustomProperty});
```

Comment:
This seems like reasonable default styling to place in the card styles itself. i.e. 

```
${display('flex')}

: host {
    flex-direction: column;
    gap: ${standardPadding};
}
```
Know why we wouldn't make that the default?

Reply:
Done!

Reply:
This gets a little more complicated when we add the `title` slot and add the `section` to the template for accessibility, but I think we can sort through that in the next PR that adds the title slot.